download/index.md

```diff
@@ -43,6 +43,11 @@ redirect_from: /licenses/flyway-trial
 <tr><td><a href="/documentation/configuration/parameters/cherryPick">Cherry picking</a></td><td></td><td><i class="fa fa-check"></i></td></tr>
 <tr><td><a href="/documentation/configuration/parameters/skipExecutingMigrations">Mark as applied</a></td><td></td><td><i class="fa fa-check"></i></td></tr>
 <tr><td><a href="/documentation/concepts/migrations#query-results">Toggle display of query results</a></td><td></td><td><i class="fa fa-check"></i></td></tr>
+<tr>
+    <td><a href="/documentation/configuration/authentication">Additional authentication mechanisms</a></td>
```

Comment:
I think this could just read "Supported Authentication mechanisms" to make it absolute (rather than relative to env vars and un/pw